src/code/z_viscvg.c

```diff
@@ -1,70 +1,103 @@
-#include "global.h"
+/**
+ * @file z_viscvg.c
+ * Description: Visualise Coverage in various ways.
+ *
+ * Coverage is roughly how much of a pixel is covered by a primitive; it is used for antialiasing, see PreRender.c and
+ * §15 of the programming manual for details.
+ *
+ * To understand this file, it is helpful to remember that A_MEM is essentially synonymous with coverage, and that
+ * `GBL_c1/2(p, a, m, b)` are usually `(p * a + m * b) / (a + b)`.
+ *
+ * Coverage is full when not on an edge, while on an edge it is usually lower, and since coverage is treated as an
+ * alpha value, edges with lower coverage will show up as darker than interiors in all of the available modes.
+ *
+ * Coverage is abbreviated to "cvg"; "pixel RGB" is the original colour the pixel had before the filter is applied.
```

Comment:
Well in what is currently named `sCoverageRGBPrimFogDL` "pixel RGB" at the second blender is already not the colour from the before-filter framebuffer, it has been lerped with primColor

Maybe just use "FB RGB" as in "frame buffer rgb"?

Reply:
also colour -> color, since we use the latter pretty much everywhere else.
a few should be fixed up in a seperate cleanup pr or something 
![image](https://user-images.githubusercontent.com/22045047/188325577-6fae54b3-ffb2-4175-90e3-bb0f7778025c.png)